### DISCUSSION TITLE
Unpacking

### DISCUSSION THREAD

**Discussion by Inferdy** [mxgmn/MarkovJunior#39] [Q&A] [answered]
Is there a way to get the right detailed 13x13 pixels scheme from the initial 3x3 representation?

![image](https://user-images.githubusercontent.com/24939153/174329219-4a34b013-9118-4d7e-b193-8372a638f72d.png)

**Comment by mxgmn** [comment, marked as answer]
You can do it with the `map` node:
```xml
<sequence values="WO" symmetry="()">
  <one in="WWW/WWW/WWW" out="***/*OO/***"/>
  <map scale="13/3 13/3 1" values="BWO">
    <rule in="W" out="BBBB/BWWW/BWWW/BWWW"/>
    <rule in="O" out="BBBB/BOOO/BOOO/BOOO"/>
  </map>
</sequence>
```

**Reply by Inferdy** [reply]
Wow, it's so cool. I wish I knew it before I started my WIP project 😂
![6k0sds](https://user-images.githubusercontent.com/24939153/174354827-1f15dc60-6161-4d67-b924-e3e37ff86446.gif)
